MonoGame.Framework/Graphics/OpenGL.MacOS.cs

```diff
@@ -7,16 +7,23 @@
 using ObjCRuntime;
 using AppKit;
 using System.Security;
+using MonoGame.Utilities;
 
 namespace OpenGL
 {
     public partial class GL
-	{
+    {
+        public static IntPtr Library = FuncLoader.LoadLibrary("/System/Library/Frameworks/OpenGLES.framework/OpenGLES");
```

Comment:
Should we only be using OpenGLES on Desktop Mac?

Reply:
I think this is a cut and paste typo.. it should be "/System/Library/Frameworks/OpenGLES.framework/OpenGL".

Reply:
https://github.com/MonoGame/MonoGame/pull/6390